File: app/src/main/java/com/grace/placessearch/data/model/Participant.java
package com.grace.placessearch.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Participant {

    @SerializedName("participant")
    @Expose
    private Participant participant;
    @SerializedName("role")
    @Expose
    private String role;

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }

}

File: app/src/main/kotlin/com/grace/placessearch/common/data/model/Participant.kt
package com.grace.placessearch.common.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Participant {

    @SerializedName("participant")
    @Expose
    var participant: Participant? = null
    @SerializedName("role")
    @Expose
    var role: String? = null

}